Dockerfile

```diff
@@ -0,0 +1,75 @@
+ARG GO_VERSION=1.17.5
+FROM golang:$GO_VERSION-bullseye as builder
+
+ARG CHANNEL=nightly
+ARG URL=
+ARG BRANCH=
+ARG SHA=
+
+# Basic dependencies.
+ENV HOME /node
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && \
+    apt-get install -y \
+    apt-utils \
+    bsdmainutils \
+    curl \
+    git \
+    git-core \
+    python3
+
+COPY ./docker/files/ /node/files
+COPY ./installer/genesis /node/files/run/genesis
+COPY ./cmd/updater/update.sh /node/files/build/update.sh
+COPY ./installer/config.json.example /node/files/build/config.json
```

Comment:
This is the only change compared to the `winder/docker` repo. The files were moved from the project root to the `./docker/files` directory, and files which already exist in this repo (genesis files, update.sh, config.json) are used directly instead of copied.